### DISCUSSION TITLE
Soup error unable to copy to /salt/common/tools/sbin/

### DISCUSSION THREAD

**Discussion by ramsri** [Security-Onion-Solutions/securityonion#4310] [Unsupported Versions]
When I tried to upgrade from 2.3.10 to 2.3.51 - CentOS
I keep on getting this error message 
error unable to copy to /salt/common/tools/sbin/

After going through the script I found that Default_Salt_Dir is not defined or missing.
not sure if this is happening to me alone or everyone.
Found that it is missing from this file as well 
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/common/tools/sbin/soup

As a workaround, I explicitly defined DEFAULT_SALT_DIR=/opt/so/saltstack/default in soup script.

After this, I was able to update.
Just want to understand the exact cause.

**Comment by EddyIAM** [comment]
Not just you.   I had the same issue trying to update a fresh install of 2.3.0.  Defining DEFAULT_SALT_DIR fixed the issue for me also.  Thanks!
